kuma/javascript/src/payments/feedback-form.jsx

```diff
@@ -0,0 +1,55 @@
+// @flow
+import * as React from 'react';
+import { gettext } from '../l10n.js';
+import GAProvider from '../ga-provider.jsx';
+
+const FeedbackForm = () => {
+    const [feedback, setFeedback] = React.useState('');
+    const [showMessage, setShowMessage] = React.useState(false);
+    const ga = React.useContext(GAProvider.context);
+
+    const handleChange = event => {
+        const { value } = event.target;
+        setFeedback(value);
+
+        // cases where user submitted feedback already
+        // and start typing in text input again
+        setShowMessage(false);
+    };
+
+    const handleSubmit = event => {
+        event.preventDefault();
+
+        ga('send', {
+            hitType: 'event',
+            eventCategory: 'monthly payments',
+            eventAction: feedback,
+            eventLabel: 'thank you page'
+            // eventValue: feedback
```

Comment:
Have an open question for Kadir about this value, which is why it's commented out at the moment.

Reply:
This has been resolved. `eventLabel` will be the feedback from the user.